torch/testing/_internal/common_methods_invocations.py

```diff
@@ -277,7 +280,22 @@ def get_operator_variant(self):
         """
         return self.operator_variant
 
-    def sample_inputs(self, device, dtype, requires_grad=False, **kwargs):
+    def conjugated_sample_inputs(self, samples):
+        """Returns an iterable of conjugated SampleInputs.
```

Comment:
More precisely: returns the sample inputs but with the tensor input or first tensor in a sequence input conjugated